Sources/Actions/Feed.php

```diff
@@ -2915,7 +2916,7 @@ public static function cdataParse(string $data, string $ns = '', bool $force = f
 				$pos = empty($positions) ? $n : min($positions);
 
 				if ($pos - $old > 0) {
-					$cdata .= substr($data, $old, $pos - $old);
+					$cdata .= substr($data, $old, (int) ($pos - $old));
```

Comment:
Casting the result of the expression to int is unnecessary. The result of the arithmetic operation will already be an integer.